src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -1064,7 +1082,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Get
         End Property
 
-        Private ReadOnly Property IConversionExpression_IsExplicit As Boolean Implements IConversionExpression.IsExplicit
+        Private ReadOnly Property IConversionExpression_Conversion As IConversion Implements IConversionExpression.Conversion
+            Get
+                Return New Conversion(ConversionKind.UserDefined, Me.Call.Method)
```

Comment:
I'll let the IOperation owners make that change.  That seems out of scope of what i'm trying to provide here.  I don't jsut want to make the bound node not implement IConversionExpression as clearly more work will need to be done past that.  I'm not familiar enough with the bound nodes and whatnot to do that work as effectively or efficiently as others will be.